libraries/wallet/wallet.cpp

```diff
@@ -1932,6 +1932,32 @@ class wallet_api_impl
       return sign_transaction( tx, broadcast );
    } FC_CAPTURE_AND_RETHROW( (owner_account)(broadcast) ) }
 
+   map<string, son_id_type> list_active_sons()
+   { try {
+           global_property_object gpo = get_global_properties();
+           std::vector<fc::optional<son_object>> son_objects =
+                   _remote_db->get_sons(gpo.active_sons);
```

Comment:
I think code of this 2 lines is short enough to make it in 1. 120 characters can be a limit that will make the code in github to display without horizontal scrolling in major browsers and screen sizes. We don't have any number defined but in this case, if you want, can consider making it in 1 line.